pkgs/desktops/cinnamon/xapps/default.nix

```diff
@@ -0,0 +1,81 @@
+{ fetchFromGitHub
+, fetchpatch
+, glib
+, gobject-introspection
+, gtk3
+, libgnomekbd
+, libxklavier
+, meson
+, ninja
+, pkgconfig
+, python3
+, python3Packages
+, stdenv
+, vala
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xapps";
+  version = "1.6.4";
+
+  src = fetchFromGitHub {
+    owner = "linuxmint";
+    repo = pname;
+    rev = version;
+    sha256 = "08c8j7wv56zf8mfgjfhd0wc1bbrw0dj1lmlawvsxwfajnnyjsm0d";
+  };
+
+  NIX_CFLAGS_COMPILE = [ "-I${glib.dev}/include/gio-unix-2.0" ]; # TODO: https://github.com/NixOS/nixpkgs/issues/36468
+
+  patches = [
+    (fetchpatch { # details see https://github.com/linuxmint/xapps/pull/65
+      url = "https://github.com/linuxmint/xapps/compare/d361d9cf357fade59b4bb68df2dcb2c0c39f90e1...2dfe82ec68981ea046345b2be349bd56293579f7.diff";
+      sha256 = "0sffclamvjas8ad57kxrg0vrgrd95xsk0xdl53dc3yivpxkfxrnk";
+    })
+  ];
+
+  buildInputs = [
+    glib
+    gtk3
+    libgnomekbd
+    libxklavier
+    (python3.withPackages(ps: with ps; [ pygobject3 ]))
```

Comment:
not needed 
```suggestion
```

Reply:
Won't that remove the python interpreter?

Reply:
@jonringer That actually is conflicting with what I suggested @mkg20001 to do.

Reply:
https://github.com/NixOS/nixpkgs/pull/75138#discussion_r355109693

Reply:
https://github.com/NixOS/nixpkgs/pull/75138#discussion_r355111509

Reply:
@mkg20001 But those are different. You had to add it so the pkgconfig file could be in `PATH`. We use the wrapped python here so when shebangs get patched they have the correct packages.